web/permissions/permissions.go

```diff
@@ -128,6 +130,17 @@ func createPermission(c echo.Context) error {
 		return err
 	}
 
+	var expiresAt *time.Time
+	if ttl != "" {
+		if d, errd := bigduration.ParseDuration(ttl); errd == nil {
+			ex := time.Now().Add(d)
+			expiresAt = &ex
+			if d.Hours() > 1.0 {
+				tiny = false
```

Comment:
Since we can not respect what the dev expect, is it possible to add a log to explain why? Even if it's written in the documentation I think it's a good to have it logged

```suggestion
     if tiny == true {
        log('Tiny can not be set to true since duration is > 1h')
     }       
     tiny = false
```